components/style/values/computed/font.rs

```diff
@@ -5,13 +5,17 @@
 //! Computed values for font properties
 
 use app_units::Au;
+use cssparser::Parser;
+use parser::{Parse, ParserContext};
 use std::fmt;
-use style_traits::ToCss;
+use style_traits::{ToCss, StyleParseErrorKind, ParseError};
 use values::animated::ToAnimatedValue;
-use values::computed::NonNegativeLength;
+use values::computed::{Context, NonNegativeLength, ToComputedValue};
 use values::specified::font as specified;
+use values::specified::length::{FontBaseSize, NoCalcLength};
 
-pub use values::specified::font::XTextZoom;
+pub use values::computed::Length as MozScriptMinSize;
```

Comment:
I think we should avoid adding `use`s like this. We might need Length in that file at some point. How about creating a type alias instead?

Reply:
This is fine, we do this in another places.

Reply:
A type alias has the inconvenience that you can't use enum variants anymore, this is a rust known issue, so I'd rather keep this for now.

Reply:
@emilio Thanks for the clarification! I didn't know about that type alias would have that inconvenience but now I learn it 😄